Plugins/Creature/Creature.cpp

```diff
@@ -1876,4 +1882,38 @@ ArgumentStack Creature::SetDisarmable(ArgumentStack&& args)
     }
     return Services::Events::Arguments();
 }
+
+ArgumentStack Creature::SetFaction(ArgumentStack&& args)
+{
+    int32_t retVal = -1;
+    if (auto *pCreature = creature(args)){
```

Comment:
Style nit: `{` and `}` on own lines. Also, generally avoiding prefixes (except `p-`) on variables in nwnx, so `factionId` or something.

Reply:
Done